Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -102,6 +108,37 @@ struct TestCommand: AsyncParsableCommand {
     )
     var skipConfigurations: [String] = []
 
+    @Flag(
+        name: [.customShort("x"), .long],
+        help: "When passed it uses xcframeworks (simulator and device) from the cache instead of frameworks (only simulator)."
+    )
+    var xcframeworks: Bool = false
```

Comment:
Maybe we can re-use
```
        @OptionGroup()
        var options: CacheOptions
```
?

Reply:
Agreed! Consistency is important across commands.

Reply:
We want to have different descriptions of those flags